src/content/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips.mdx

```diff
@@ -25,19 +27,15 @@ Because of this, ensure your firewall allows their network requests through it.
 
 ## IP addresses are not personal data [#not-personal-data]
 
-Minions are deployed on servers, and the agents are expected to be activated using non-personal credentials. IP addresses associated with minions running on servers are not personal data under data protection and privacy laws. For more information, see the [Synthetic's security documentation](/docs/synthetics/new-relic-synthetics/getting-started/security-new-relic-synthetics).
+Minions are deployed on servers, and the agents are activated using non-personal credentials. IP addresses associated with minions running on servers are not personal data under data protection and privacy laws. For more information, see the [Synthetic's security documentation](/docs/synthetics/new-relic-synthetics/getting-started/security-new-relic-synthetics).
 
 ## Daily JSON listings for IP addresses [#dynamic-ip-addresses]
 
 IP addresses for released locations are subject to change. If a change is needed, we'll attempt to proactively notify customers prior to any changes via e-mail. You can also check the [Explorers Hub](https://discuss.newrelic.com/) for updates.
 
 <Callout variant="important">
-   Synthetics is [adding new IP addresses on July 15 2021](/docs/synthetics/synthetic-monitoring/administration/new-synthetic-public-minion-ips).
-</Callout>
-
-<Callout variant="tip">
-  In the S3 URL paths for this feature, `production` represents US-based accounts and `eu` represents [EU-based accounts](/docs/using-new-relic/welcome-new-relic/get-started/our-eu-us-region-data-centers).
-</Callout>
+    By the end of March 2022, [new IP address ranges](/docs/synthetics/synthetic-monitoring/administration/new-synthetic-public-minion-ips) may require that you reconfigure your firewall settings.
```

Comment:
I'd also be a bit more specific about what would entail "may" becoming "will". I'd specify in which instances someone would have to reconfigure.

Reply:
"May" was to account for the short period that both runtimes would be supported for already-existing monitors. Users won't have to update their allow lists until:

- They create a new monitor after the new runtime launch
- The old runtime (and their associated / old IPs) is no longer supported. (No date for this yet) 

At that point, they'll have to update their allow lists. The call out is mostly just to notify users that this is in the works to keep an eye out for.